app/routers/audio.py

```diff
@@ -0,0 +1,475 @@
+        else:
+            CHOSEN_SFX = str(pathlib.Path(SOUNDS_PATH).joinpath(sfx_choice))
+            CHOSEN_VOL = sfxs_vol
+
+        if sfxs_on == "On" or sfxs_on is None:
+            sfx_thread = threading.Thread(target=init_sfx, daemon=True)
+            sfx_thread.start()
+
+    return RedirectResponse("/", status_code=HTTP_302_FOUND)
+
+
+@router3.get("/")
+async def stop_audio() -> RedirectResponse:
+    """Stops audio.
+
+    Returns:
+        RedirectResponse: redirect the user to home.html.
+    """
+    if mixer.get_init():
+        mixer.stop()
+        global LISTENER
+        LISTENER.stop()
+        LISTENER = Listener(on_click=on_click)
+
+    return RedirectResponse("/", status_code=HTTP_302_FOUND)
+
+
+# Functions for audio control:
+
+
+def init_audio_tracks(
+        session: Session,
+        song_names: List[str],
+        sfx_names: List[str]):
+    """This function fills the AudioTracks table
+
+    Args:
+        session (Session): the database
+        song_names (List[str]): list of song names
+        sfx_names (List[str]): list of sfxs' names.
+    """
+    for song in song_names:
+        add_audio_track(session, song, True)
+    for sfx in sfx_names:
+        add_audio_track(session, sfx, False)
+
+
+def add_audio_track(session: Session, title: str, is_music: bool):
+    """Adds a new audio track to AudioTracks table.
+
+    Args:
+        session (Session): the databse.
+        title (str): title of the track.
+        is_music (bool): if set to true - music, sfx otherwise.
+    """
+    res = session.query(AudioTracks).filter_by(title=title).first()
+    if not res:
+        track = AudioTracks(title=title, is_music=is_music)
+        session.add(track)
+        session.commit()
+
+
+def play_music(tracks: List[str], vol: int):
+    """This function gets a music playlist and a desired volume as input.
+    It loops through the list until it stops by the users' input.
+
+    Args:
+        tracks (List[str]): list of music tracks.
+        vol (int): chosen volume for the tracks.
+    """
+    while mixer.get_init():
+        for track in tracks:
+            track = SOUNDS_PATH + track
+            music = mixer.Sound(track)
+            music.set_volume(vol)
+            music.play()
+            sleep(music.get_length())
+
+
+def init_sfx():
+    """This function starts listening for mouse clicks on a new thread.
+    On each click it calls the on_click function,
+    which plays a sound effect chosen by the user.
+    """
+    LISTENER.start()
+
+
+def on_click(_, _1, button: Button, pressed: bool):
+    """For each left-button mouse click, plays a sound effect
+    chosen by the user in the desired volume.
+
+    Args:
+        button (Button): a mouse button object.
+        pressed (bool): a bool indicating if a mouse button was pressed.
+    """
+    if pressed and button == button.left:
+        sfx = mixer.Sound(CHOSEN_SFX)
+        sfx.set_volume(CHOSEN_VOL)
+        sfx.play()
```

Comment:
This code will run on the server's machine. 
The user won't hear the `.play`.

You have to use JS if you want to run the code on the client side.